### ISSUE TITLE
Keep track of --legacy-690lc

### ISSUE THREAD

**Issue by jonasmalacofilho** [liquidctl/liquidctl#67]
Mixing calls with and without `--legacy-690lc` can cause issues, and having the user keep track of this is prone to errors.

One solution would be for the value – `legacy_690lc = True | False` – passed to `initialize` to persist until either the device is initialized again, or the system is rebooted.

```
## default to modern/EVGA

# liquidctl list
Device ID 0: Asetek 690LC (assuming EVGA CLC)
# liquidctl status
Asetek 690LC (assuming EVGA CLC)
├── Liquid temperature        29.7  °C
├── Fan speed                  480  rpm
├── Pump speed                1890  rpm
└── Firmware version      2.10.0.0  


## set to legacy/NZXT

# liquidctl initialize --legacy-690lc
# liquidctl list
Device ID 0: Asetek 690LC (assuming NZXT Kraken X)
# liquidctl status
Asetek 690LC (assuming NZXT Kraken X)
├── Liquid temperature        29.7  °C
├── Fan speed                  480  rpm
├── Pump speed                1890  rpm
└── Firmware version      2.10.0.0  


## revert back to default

# liquidctl initialize
# liquidctl list
Device ID 0: Asetek 690LC (assuming EVGA CLC)
```

I'm still not sure what should be the behavior when `--legacy-690lc` is passed to commands other than `initialize`.

---

The implementation could check for the existence of the temporary data files used by the legacy driver.

But this would change some `probe` code, and it might be too late to do that in 1.3.0. 
 These new checks outside of the legacy driver could cause some unforeseen issues (e.g. life-cycle issues, or permission problems for those using udev rules).

---

Possibly related: #66

**Comment by MarshallAsch** [comment]
Would it work to store this value in the `keyval.py` key value storage system? I believe that is backed by a file that persists until the system is rebooted.

**Comment by jonasmalacofilho** [comment]
Yes it would, and that was my original idea.  But this could also be implemented with #195, and that's probably better: it's more explicit/persistent, and also easier to implement.
